apiserver/facades/agent/uniter/networkinfo.go

```diff
@@ -60,7 +60,7 @@ func NewNetworkInfo(st *state.State, tag names.UnitTag) (NetworkInfo, error) {
 	return n, errors.Trace(err)
 }
 
-// NewNetworkInfoWithBehaviour initialises and returns a new NetworkInfo
+// NewNetworkInfoForStrategy initialises and returns a new NetworkInfo
```

Comment:
While you're here, the comment block has no explanation of the retryFactory arg.

Reply:
It is assigned directly to a member which has I think, a cogent comment.